stable/kong/values.yaml

```diff
@@ -0,0 +1,268 @@
+# Default values for kong.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: kong
+  tag: 0.11.2
+  pullPolicy: IfNotPresent
+
+# Specify Kong admin and proxy services configurations
+admin:
+  # HTTPS traffic on the admin port
+  https:
+    servicePort: 8444
+    containerPort: 8444
+  http:
+    servicePort: 8001
+    containerPort: 8001
+  # Kong admin service type
+  type: NodePort
+  nodePort: 32444
+proxy:
+  # HTTPS traffic on the proxy port
+  https:
+    servicePort: 8443
+    containerPort: 8443
+  http:
+    servicePort: 8000
+    containerPort: 8000
+  type: NodePort
+  nodePort: 32443
+
+# Set runMigrations to run Kong migrations
+runMigrations: true
+
+# Specify Kong configurations
+# Kong configurations guide https://getkong.org/docs/latest/configuration/
+env:
+  database: postgres
+  pg_password: kong
```

Comment:
You should not added a second field for the DB password. There is already `postgresql.postgresPassword`. Also, by default, a random password is generated. Let's keep it that way.